tests/drivers/can/timing/src/main.c

```diff
@@ -294,11 +294,6 @@ void test_main(void)
 	printk("testing on device %s @ %u Hz\n", dev->name, core_clock);
 
 	k_object_access_grant(dev, k_current_get());
-
-	ztest_test_suite(can_timing_tests,
-			 ztest_user_unit_test(test_set_timing_min),
-			 ztest_user_unit_test(test_set_timing_max),
-			 ztest_user_unit_test(test_timing),
-			 ztest_user_unit_test(test_timing_data));
```

Comment:
These are all userspace tests, but you migrated them to non-userspace tests. Please fix.

Reply:
sorry, let me check